core/src/main/java/org/apache/carbondata/core/datastore/block/SegmentProperties.java

```diff
@@ -54,6 +54,8 @@
    */
   private KeyGenerator dimensionKeyGenerator;
 
+  private KeyGenerator sortColumnsGenerator;
```

Comment:
please add comment for this member variable and `dimensionKeyGenerator`, to describe their difference